mlflow/gateway/constants.py

```diff
@@ -2,3 +2,7 @@
 MLFLOW_GATEWAY_CRUD_ROUTE_BASE = "/api/2.0/gateway/routes/"
 MLFLOW_GATEWAY_ROUTE_BASE = "/gateway/"
 MLFLOW_QUERY_SUFFIX = "/invocations"
+
+# Provider constants
+MLFLOW_AI_GATEWAY_ANTHROPIC_MAX_TOKENS = 1000000
+MLFLOW_AI_GATEWAY_ANTHROPIC_DEFAULT_TOKENS = 200000
```

Comment:
```suggestion
MLFLOW_AI_GATEWAY_ANTHROPIC_MAX_TOKENS = 1_000_000
MLFLOW_AI_GATEWAY_ANTHROPIC_DEFAULT_TOKENS = 200_000
```

Reply:
Does Anthropic support 1M max_tokens?

Reply:
yep! Validated that it supports up to exactly 1M. Anything above this throws an cryptic error from their service.
![Screenshot 2023-06-30 at 1 38 04 PM](https://github.com/mlflow/mlflow/assets/39283302/0ca888e0-1279-4546-9188-3629c3543b29)

Reply:
What if we use `1_000_000 + 1`?

Reply:
![too many tokens anthropic](https://github.com/mlflow/mlflow/assets/39283302/68e0e321-685c-49a6-af9f-d3e9d5e2812a)

Reply:
(this is without the limit check handling in this PR)

Reply:
Maybe they implemented https://arxiv.org/abs/2304.11062

Reply:
They announced 100K Context Windows about two months ago:

https://www.anthropic.com/index/100k-context-windows